build.sh

```diff
@@ -212,6 +212,9 @@ fi
 if [[ -z ${STRIP_DEBUG_INFO} ]]; then
     STRIP_DEBUG_INFO=OFF
 fi
+if [[ -z ${NO_MEM_TRACKER} ]]; then
+    NO_MEM_TRACKER=OFF
```

Comment:
USE_MEM_TRACKER=OFF

Reply:
Yes, I changed to `USE_MEM_TRACKER` in pr: https://github.com/apache/incubator-doris/pull/9661